docs/Measuring_Resonances.md

```diff
@@ -207,7 +207,7 @@ software dependencies not installed by default. First, run on your Raspberry Pi
 the following commands:
 ```
 sudo apt update
-sudo apt install python3-numpy python3-matplotlib libatlas-base-dev
+sudo apt install python3-numpy python3-matplotlib libatlas-base-dev libopenblas
```

Comment:
See https://packages.debian.org/bullseye/libopenblas-base

The correct name is `libopenblas-base`

Reply:
My bad. Need more coffee.